src/logcollector/read_win_event_channel.c

```diff
@@ -170,7 +182,7 @@ char *get_message(EVT_HANDLE evt, LPCWSTR provider_name, DWORD flags)
         goto cleanup;
     }
 
-    if ((buffer = calloc(size, sizeof(wchar_t))) == NULL) {
+    if ((buffer = calloc(++size, sizeof(wchar_t))) == NULL) {
```

Comment:
To improve code readability I suggest changing this line as follows:
 ```suggestion
    // Buffer size is increased by one unit as `EvtFormatMessage` does not count the '\0'
    size += 1;
    if ((buffer = calloc(size, sizeof(wchar_t))) == NULL) {
```